tests/api/test_cluster.cpp

```diff
@@ -56,6 +58,39 @@ void setup_wormhole_remote(Cluster* umd_cluster) {
 // This test should be one line only.
 TEST(ApiClusterTest, OpenAllChips) { std::unique_ptr<Cluster> umd_cluster = get_cluster(); }
 
+TEST(ApiClusterTest, DifferentConstructors) {
+    std::vector<int> pci_device_ids = PCIDevice::enumerate_devices();
+    // TODO: Make this test work on a host system without any tt devices.
+    if (pci_device_ids.empty()) {
+        GTEST_SKIP() << "No chips present on the system. Skipping test.";
+    }
```

Comment:
I like this guard of stopping run of silicon tests without silicon, can we somehow add it to all tests that we are running?